server/ladder_service.py

```diff
@@ -410,6 +422,11 @@ def get_player_mean(player):
             mapname = re.match("maps/(.+).zip", map_path).group(1)
             # FIXME: Database filenames contain the maps/ prefix and .zip suffix.
             # Really in the future, just send a better description
+
+            additional_timeout = 0
+            if "neroxis_map_generator" in mapname:
+                additional_timeout += 30
+
```

Comment:
IMO, just increase the base timeout to 60.

Reply:
Aight